config_files/alertmanager/config.yaml

```diff
@@ -3,10 +3,14 @@ route:
   group_wait: 30s
   group_interval: 5m
   repeat_interval: 3h
-  receiver: discord
+  receiver: alerts
 
 receivers:
-- name: discord
+- name: alerts
```

Comment:
Why do we have this name changed?

Reply:
According to this part:
```
receivers:
- name: alerts
  discord_configs:
  - webhook_url: <your_discord_webhook>
  slack_configs:
  - send_resolved: true
    text: "{{ .CommonAnnotations.description }}"
    channel: <#your_slack_channel_name>
    api_url: <your_slack_webhook>
    ```
  Now we have two receivers and in the configuration, you can only put one name as the `receiver` in `route` so I change it to a general name.

Reply:
Yes, you are right. Thanks!